scripts/inputs/tasks.csv

```diff
@@ -961,3 +961,4 @@ Id,Name,Description,Difficulty,Type,Subcategory,Skill requirements,Area,Addition
 957,Equip an Elysian Spirit Shield ,Equip an Elysian Spirit Shield. ,Master,Bossing,Corporeal beast,75 Defence (ub); 85 Prayer (ub),Wilderness,
 958,Obtain Every Revenant Weapon ,Obtain Craw's Bow Viggora's Chainmace and Thammaron's Sceptre as drops from Revenants.,Master,Combat,Revenants,,Wilderness,
 959,Equip an Amulet of Eternal Glory ,Equip an amulet of eternal glory.,Master,Other,Other,,Wilderness,Fremennik OR Asgarnia
+960,Trailblazer Boon ,Receive a free 250 points for re-opening a League Menu. See game update newspost on 09/12/20 for more information.,Elite,Other,Other,,Common,
```

Comment:
I wasn't sure what to use as the task ID, since the one output by the Runelite plugin conflicts with an existing task (734).